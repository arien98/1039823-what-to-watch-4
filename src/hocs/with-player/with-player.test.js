import React from "react";
import renderer from "react-test-renderer";
import withVideo from "./with-player.js";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withVideo(MockComponent);

describe(`with player snapshot`, () => {
  test(`it renders correctly`, () => {
    const props = {
      filmData:
        {
          id: 2,
          title: `Bohemian Rhapsody`,
          poster: `img/bohemian-rhapsody.jpg`,
          preview: ``,
        },
    };

    const tree = renderer
      .create(<MockComponentWrapped {...props}></MockComponentWrapped>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
