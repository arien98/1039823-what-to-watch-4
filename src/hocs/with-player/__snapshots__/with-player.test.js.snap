// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with player snapshot it renders correctly 1`] = `<div />`;
